lib/pengine/clone.c

```diff
@@ -975,7 +975,8 @@ pe__clone_text(pcmk__output_t *out, va_list args)
             // Print individual instance when non-probe action is pending
             print_full = TRUE;
 
-        } else if (partially_active == FALSE) {
+        } else if (partially_active == FALSE
+                   && pcmk_is_set(child_rsc->flags, pe_rsc_managed)) {
```

Comment:
If an instance is inactive, the current behavior is fine -- it's only shown if inactive resources are requested to be shown, regardless of whether it's managed or not. What would be interesting is an active, unmanaged clone instance on an offline node.